components/clearbutton/index.css

```diff
@@ -80,9 +59,8 @@
 .spectrum-ClearButton {
 	block-size: var(--mod-clear-button-height, var(--spectrum-clear-button-height));
 	inline-size: var(--mod-clear-button-width, var(--spectrum-clear-button-width));
-	border-radius: 100%;
 
-	background-color: var(--mod-clear-button-background-color, transparent);
+	background-color: transparent;
```

Comment:
Nice, let's make sure this gets documented clearly in the changelog that this mod no longer works here.